src/OrbitQt/CaptureOptionsDialog.cpp

```diff
@@ -30,56 +30,67 @@ CaptureOptionsDialog::CaptureOptionsDialog(QWidget* parent)
 
   QObject::connect(ui_->buttonBox, &QDialogButtonBox::accepted, this, &QDialog::accept);
   QObject::connect(ui_->buttonBox, &QDialogButtonBox::rejected, this, &QDialog::reject);
-  QObject::connect(ui_->unwindingMethodComboBox, qOverload<int>(&QComboBox::currentIndexChanged),
-                   ui_->maxCopyRawStackSizeLineEdit, [this]() {
-                     auto unwinding_method = ui_->unwindingMethodComboBox->currentData().toInt();
-                     ui_->maxCopyRawStackSizeLineEdit->setEnabled(unwinding_method ==
-                                                                  CaptureOptions::kFramePointers);
+  QObject::connect(ui_->framePointerUnwindingRadioButton, qOverload<bool>(&QRadioButton::toggled),
+                   ui_->maxCopyRawStackSizeWidget,
+                   [this](bool checked) { ui_->maxCopyRawStackSizeWidget->setEnabled(checked); });
+  QObject::connect(ui_->collectMemoryInfoCheckBox, qOverload<bool>(&QCheckBox::toggled),
+                   ui_->memoryHorizontalLayout, [this](bool checked) {
```

Comment:
Nit: To be on the safe side `ui_->memoryHorizontalLayout` should be replaced by `this`. `ui_->memorySamplingPeriodMsLabel` and `ui_->memorySamplingPeriodMsLineEdit`, etc. are child elements of the `CaptureOptionsDialog`. So this connection should get disconnected as soon as the CaptureOptionsDialog gets into deconstruction.